datahub_sap_hana/ingestion.py

```diff
@@ -178,3 +200,115 @@ def _get_view_lineage_workunits(self) -> Iterable[MetadataWorkUnit]:
                 wu = item.as_workunit()
                 self.report.report_workunit(wu)
                 yield wu
+
+    def get_column_lineage_view_definitions(self) -> Iterable[Tuple[str, str]]:
+
+        url = self.config.get_sql_alchemy_url()
+        engine = create_engine(url, **self.config.options)
+
+        with engine.connect() as conn:
+            inspector = inspect(conn)
+            schema = inspector.get_schema_names()  # returns a list
+            views = inspector.get_view_names(
+                schema[0])  # returns a list
+
+            for view in views:
+                definition = inspector.get_view_definition(view, schema[0])
+                yield (view, definition)
+                # print(view, definition)
+
+    def get_lineage_for_column(self, view_name: str, view_sql: str) -> Iterable[Tuple[str, Any]]:
+
+        cols = parse_one(view_sql).named_selects
+        for col_name in cols:
+            yield (view_name, lineage(col_name.lower(), view_sql.lower()))
```

Comment:
i think it might be easier to not use `yield` here and instead build an actual list of lineages

```suggestion
        view_sql = view_sql.lower()
        lineages = [ 
           lineage(col_name.lower), view_sql)
           for col_name in cols
        ]
        return view_name, lineages
```

It might even make sense to change how much lineage is returned, you could do something like this because we _only_ really care about the first downstream list, we don't need the whole lineage node do we?

```
        view_sql = view_sql.lower()
        lineages = [ 
           (col_name.lower, lineage(col_name.lower), view_sql).downstream)
           for col_name in cols
        ]
        return view_name, lineages
```

this gives you return type that looks like `Tuple[str, List[str, List[Node]]]`

meaning it looks like 

```
(
    view_name,
    [
      (
          column_one, 
          [node1, node2]
      ),
      (
         column_two,
         [another_node]
      )
    ]
)
```

The benefit of this output is that the column lineage stays grouped by the view and you only get the data that you need, the first step in the lineage for each column.